client/src/pages/Search/MapView/RestaurantPreview.js

```diff
@@ -1,7 +1,84 @@
+import React from "react";
+import ReviewStars from "../../../components/ReviewStars";
+import { IconButton } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import { Link } from "react-router-dom";
+import YelpLogo from "../../../assets/images/yelp-logo.png";
+import RedirectButton from "../../../components/RedirectButton";
+
+const classes = {
+  previewContainer: {
+    height: "160px",
+    width: "100%",
+    backgroundColor: "rgb(224,224,224)",
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  imageContainer: {
+    height: "100%",
+    width: "100px",
+    marginLeft: "1rem",
+    objectFit: "cover",
+  },
+  detailsContainer: {
+    marginLeft: "10px",
+  },
+  nameContainer: {
+    fontSize: "1.5rem",
+    textDecoration: "none",
+    color: "black",
+  },
+  categoriesContainer: {
+    marginTop: "4.8rem",
+  },
+  exitContainer: {
+    marginLeft: "6rem",
+  },
+  yelpButtonContainer: {
+    marginTop: "3.2rem",
+  },
+};
+
 function RestaurantPreview({ restaurant }) {
+  const restaurantPage = "/restaurant?yelp_id=" + restaurant.yelp_id;
+  const restaurantAddress =
+    restaurant.address.address1 +
+    ", " +
+    restaurant.address.state +
+    ", " +
+    restaurant.address.zip_code;
+
   return (
-    <div style={{ height: "160px", width: "100%", backgroundColor: "gray" }}>
-      {restaurant.name}
+    <div style={classes.previewContainer}>
+      <div style={{ display: "flex" }}>
+        <img
+          style={classes.imageContainer}
+          src={restaurant.image_url}
+          alt={restaurant.image_url}
+        />
+        <div style={classes.detailsContainer}>
+          <Link style={classes.nameContainer} to={restaurantPage}>
+            {restaurant.name}
+          </Link>
+          <div>
+            <ReviewStars rating={restaurant.rating} />
+          </div>
+          <div>{restaurantAddress}</div>
+          <div style={classes.categoriesContainer}>
+            {restaurant.food_categories.join(" • ")}
+          </div>
+        </div>
+      </div>
+      <div>
+        <div>
+          <IconButton style={classes.exitContainer}>
+            <CancelIcon />
+          </IconButton>
```

Comment:
1. Have it so that the preview closes when the user clicks on this button
2. Use position absolute here because this button should always be on the top right, regardless of the position of other items in this preview card